src/components/action-toolbar/action-toolbar-button/__spec__.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import ActionToolbar from './../action-toolbar';
+import ActionToolbarButton from './action-toolbar-button';
+import Button from './../../button';
+import { mount } from 'enzyme';
+
+describe('ActionToolbarButton', () => {
+  let wrapper, button;
+  let selected = {};
+  const onClick = jest.fn();
+
+  beforeEach(() => {
+    wrapper = mount(
+      <ActionToolbar
+        actions={[]}
```

Comment:
This should be an object
https://github.com/Sage/carbon/blob/c624df3e5e9d16d23915fbb3301134536f960407/src/components/action-toolbar/action-toolbar.js#L47

Reply:
I think the action-toolbar is wrong - it should be an array of objects right? Maybe we can fix that in this PR too?